src/genophenocorr/preprocessing/_uniprot.py

```diff
@@ -20,6 +20,11 @@ def __init__(self):
         """Constructs all necessary attributes for a UniprotProteinMetadataService object
         """
         self._logger = logging.getLogger(__name__)
+        self._logger.setLevel(logging.INFO)
```

Comment:
This logging configuration must be removed because setting up logging is the user's responsibility, not ours.

Note, you can use this to quickly setup logging in notebook or a script:

```python
import hpotk
hpotk.util.setup_logging()
```